validator/src/cli.rs

```diff
@@ -2775,6 +2775,12 @@ pub fn test_app<'a>(version: &'a str, default_args: &'a DefaultTestArgs) -> App<
                 .validator(is_parsable::<u64>)
                 .takes_value(true)
                 .help("Override the runtime's account lock limit per transaction"),
+        )
+        .arg(
+            Arg::with_name("enable_bigtable_ledger_upload")
+                .long("enable-bigtable-ledger-upload")
+                .takes_value(false)
+                .help("Enable ledger upload to BigTable"),
```

Comment:
```suggestion
        )
        .arg(
            Arg::with_name("enable_bigtable_ledger_upload")
                .long("enable-bigtable-ledger-upload")
                .takes_value(false)
                .help("Upload new confirmed blocks into a BigTable instance"),
```
For consistency with the solana-validator arg

Reply:
Done :)